### PR TITLE
added deleted files

### PR DESCRIPTION
**Pull request by Pauatro**

### CODE REVIEW

File: staff/pau-atienza/precourse/Calculadora-Pro.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+function calculatorPro() {
+    //We declare arguments as a variable
+    var args = arguments; 
+    //We declare the four main operations as function expressions (could also just be functions)
+    //Addition
+    const addition = function() {
+        let add = args[0];
+        for ( let i = 1; i < args.length; i++) {
```
Comment (sergiborja) [inline staff/pau-atienza/precourse/Calculadora-Pro.js:8]: Let!!! Que es eso!?

File: staff/pau-atienza/precourse/calculadora.HTML/index.js [hunk 2]
```diff
@@ -0,0 +1,244 @@
+const one = document.querySelector('.one');
```
Comment (luislourencos) [inline staff/pau-atienza/precourse/calculadora.HTML/index.js:1]: Change 'const' to 'var'!

File: staff/pau-atienza/precourse/Calculadora-Pro.js (hunk 1 above)
Comment (Pauatro) [reply staff/pau-atienza/precourse/Calculadora-Pro.js:8]: :'(

Comment (Pauatro) [reply staff/pau-atienza/precourse/Calculadora-Pro.js]: sorted out

File: staff/pau-atienza/precourse/calculadora.HTML/index.js (hunk 2 above)
Comment (Pauatro) [reply staff/pau-atienza/precourse/calculadora.HTML/index.js]: done

**Pull request merged**